grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/JwtParser.java

```diff
@@ -73,5 +80,14 @@ private DefaultJwt(DecodedJWT jwt) {
     public Optional<String> getEmail() {
       return Optional.ofNullable(jwt.getClaim(EMAIL_CLAIM).asString());
     }
+
+    @Override
+    public Set<String> getRoles() {
+      List<String> roles = jwt.getClaim(ROLES_CLAIM).asList(String.class);
+      if (roles == null || roles.isEmpty()) {
+        return Collections.emptySet();
+      }
+      return new HashSet<>(roles);
```

Comment:
Can we use a predictable iteration set (or a list)

Reply:
Done